mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -61,8 +61,9 @@ sealed trait CompletionValue:
         s"${printer.completionSymbol(ext.symbol)} (extension)"
       case so: CompletionValue.Symbolic =>
         printer.completionSymbol(so.symbol)
-      case CompletionValue.NamedArg(_, tpe) =>
-        printer.tpe(tpe)
+      case CompletionValue.NamedArg(label, tpe, _) =>
+        if label.startsWith("Autofill") then ""
```

Comment:
I prefer to have another `CompletionValue` subclass for the autofill-named-arguments feature.

This feature is kind of `NamedArg`, but the content is quite different from `NamedArg`, so I think there's no point representing this completion as `NamedArg` that could be error-prone in the future.

What do you think about this?

Reply:
That's a good idea, I will change it